turtlebot4_openai_tutorials/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>turtlebot4_openai_tutorials</name>
+  <version>0.0.0</version>
+  <description>Tutorials on using OpenAI to control Turtlebot 4</description>
+  <maintainer email="rgariepy@clearpath.ai">rgariepy</maintainer>
+  <license>Apache2</license>
+
+  <test_depend>ament_copyright</test_depend>
+  <test_depend>ament_flake8</test_depend>
+  <test_depend>ament_pep257</test_depend>
+  <test_depend>python3-pytest</test_depend>
+
+  <depend>rclpy</depend>
+  <depend>std_msgs</depend>
+  <depend>turtlebot4_navigation</depend>
+
+  <exec_depend>python-openai</exec_depend>
```

Comment:
Rosdep key is `python3-openai-pip`

Reply:
If it a pip rosdep, the package won't be able to be bloomed.